cmd/podman/parse.go

```diff
@@ -816,3 +816,12 @@ func getLoggingPath(opts []string) string {
 	}
 	return ""
 }
+
+// validateFileName returns an error if filename container ":"
```

Comment:
nit.  I think you mean "the filename contains" rather than "filename container".

Reply:
Fixed.